jOOQ-manual/src/main/resources/org/jooq/web/manual-3.10.xml

```diff
@@ -14462,6 +14462,10 @@ result.forEach((Object[] entities) -> {
 
                org.jooq.util.xml.XMLDatabase
 
+           This value can be used to reverse-engineer schemas defined by SQL files (requires jooq-meta-extensions dependency)
+
+               org.jooq.meta.extensions.ddl.DDLDatabase
```

Comment:
In version 3.10, the class was located in `org.jooq.util.ddl.DDLDatabase`. While we're at it, I'll also add a reference to the `JPADatabase`